lockup/LockupLinearCurvesCreator.sol

```diff
@@ -0,0 +1,219 @@
+        params.totalAmount = totalAmount; // Total amount is the amount inclusive of all fees
+        params.token = DAI; // The streaming token
+        params.cancelable = true; // Whether the stream will be cancelable or not
+        params.transferable = true; // Whether the stream will be transferable or not
+        params.broker = Broker(address(0), ud60x18(0)); // Optional parameter for charging a fee
+
+        // Setting an initial and a cliff unlock amount of 25 DAI
+        LockupLinear.UnlockAmounts memory unlockAmounts = LockupLinear.UnlockAmounts({ start: 25e18, cliff: 25e18 });
+        LockupLinear.Durations memory durations = LockupLinear.Durations({
+            cliff: 50 days, // Setting a cliff of 50 days
+            total: 100 days // Setting a total duration of 100 days
+         });
+
+        // Create the LockupLinear stream using a function that sets the start time to `block.timestamp`
+        streamId = LOCKUP.createWithDurationsLL(params, unlockAmounts, durations);
+    }
+
+    /// @dev For this function to work, the sender must have approved this dummy contract to spend DAI.
+    function createStream_ConstantLinear() public returns (uint256 streamId) {
+        // Declare the total amount as 100 DAI
+        uint128 totalAmount = 100e18;
+
+        // Transfer the provided amount of DAI tokens to this contract
+        DAI.transferFrom(msg.sender, address(this), totalAmount);
+
+        // Approve the Sablier contract to spend DAI
+        DAI.approve(address(LOCKUP), totalAmount);
+
+        // Declare the params struct
+        Lockup.CreateWithDurations memory params;
+
+        // Declare the function parameters
+        params.sender = msg.sender; // The sender will be able to cancel the stream
+        params.recipient = address(0xCAFE); // The recipient of the streamed tokens
+        params.totalAmount = totalAmount; // Total amount is the amount inclusive of all fees
+        params.token = DAI; // The streaming token
+        params.cancelable = true; // Whether the stream will be cancelable or not
+        params.transferable = true; // Whether the stream will be transferable or not
+        params.broker = Broker(address(0), ud60x18(0)); // Optional parameter for charging a fee
+
+        LockupLinear.UnlockAmounts memory unlockAmounts = LockupLinear.UnlockAmounts({ start: 0, cliff: 0 });
+        LockupLinear.Durations memory durations = LockupLinear.Durations({
+            cliff: 25 days, // Setting a cliff of 25 days
+            total: 100 days // Setting a total duration of 100 days
+         });
+
+        // Create the LockupLinear stream using a function that sets the start time to `block.timestamp`
+        streamId = LOCKUP.createWithDurationsLL(params, unlockAmounts, durations);
+    }
+
+    /// @dev For this function to work, the sender must have approved this dummy contract to spend DAI.
+    function createStream_UnlockConstant() public returns (uint256 streamId) {
```

Comment:
`createStream_UnlockConstantLinear`. The previous function is `createStream_ConstantLinear`.